pkg/model/serializer.go

```diff
@@ -757,27 +775,17 @@ func (d *Dev) UnmarshalYAML(unmarshal func(interface{}) error) error {
 				"volumes",
 			}
 
-			type hybridModeInfo struct {
-				Workdir     string            `json:"workdir,omitempty" yaml:"workdir,omitempty"`
-				Selector    Selector          `json:"selector,omitempty" yaml:"selector,omitempty"`
-				Forward     []forward.Forward `json:"forward,omitempty" yaml:"forward,omitempty"`
-				Environment Environment       `json:"environment,omitempty" yaml:"environment,omitempty"`
-				Command     Command           `json:"command,omitempty" yaml:"command,omitempty"`
-				Reverse     []Reverse         `json:"reverse,omitempty" yaml:"reverse,omitempty"`
-				Mode        string            `json:"mode,omitempty" yaml:"mode,omitempty"`
-
-				UnsupportedFields map[string]interface{} `yaml:",inline" json:"-"`
-			}
-
-			hybridModeDev := &hybridModeInfo{}
-			err := unmarshal(hybridModeDev)
-			if err != nil {
-				return err
-			}
+			for key := range hybridModeDev.UnsupportedFields {
+				isSupported := true
+				for _, unsupportedFieldName := range hybridUnsupportedFields {
+					if key == unsupportedFieldName {
+						isSupported = false
+						break
+					}
+				}
 
-			for _, field := range hybridUnsupportedFields {
-				if hybridModeDev.UnsupportedFields[field] != nil {
-					oktetoLog.Warning("In hybrid mode, the field '%s' specified in your manifest is ignored", field)
+				if !isSupported {
```

Comment:
@andreafalzetti @jLopezbarb WDYT about show just one warning that contains all the unsupported fields used instead of repeat same message for each field?

Reply:
Yes, we can do something like the message that we have for compose.

Reply:
Now that you guys mentioned it, and I also thought it was a good idea, I've done the change ✅